### ISSUE TITLE
[Discussion] Market Sale Discount vs Beta Tester Discount

### ISSUE THREAD

**Issue by Getrad** [rlguy/Blender-FLIP-Fluids#305]
@rlguy 
Just pointing out that the 20% discount for beta testing is a bit of a joke when Blender Market gives a 25% discount a week later.    It amounts to a couple of bucks, but certainly doesn't reflect well on the developer or the market.     Not impressed.

**Comment by rlguy** [comment]
Hello Getrad,

I can see how you feel about this and understand your frustrations. We had set and announced (March 15th) the beta tester discount rates long before we knew about the Blender Market Spring Sale. We do not coordinate with the Blender Market for when their sales happen and I would say this timing is unfortunate with the market sale beginning just two weeks after the FLIP Fluids release.

In hindsight, I wish that I'd have updated the beta tester discount rate to 25% when I first heard about the market sale. But I did not, and for that I apologize to anyone that is not satisfied with this issue and I take full responsibility. In the future, I will do my best to step into the mind of the customer and consider how they would feel when we make decisions to offer future discounts.

I will be leaving this issue open for discussion and anyone is free to voice their concerns.

Sincerest Apologies,
Ryan
